src/images/Yellow-Splatter-Bkgrd.svg

```diff
@@ -0,0 +1,9 @@
knZXFhGmPYIMCDHgRxmALE0lKd4mtrvMG9Fh+kqCSViHLZlbU9SinWP8zeT0ZOUzStu9h6JRCEMOquw+5rKCNI0y4MhQn2cAYhaV3gnGrN5b059y/h/7mEjoIoSMzyFg0kZ9jJJ9Y3DRyrz/hOen0/QxjPMPEVe6WSFTew4gIav2xIQc/NO8mxhBJxNGQg+4EjKR5EOMkhrOVloU/btsfVbN8Sj86LDlcaDS1Jm9BqM8il270I2zMVUrAFWL0Ntp2zwitE+jTTQy/DiSWWu6htt9C74ksnkr6nbfUpjG85qBVDfVZ3LTHKjD+lc8ms4OKNZiuJE7+lUDdh06N068adENsZT28r5P7n9G+e7Q9x4zcsI3DpBm4lxbymuM7f8nzLiyWD/dtT1FTUPSH/O0xuifgs61hMCcWefA0nVyD0fkzBqbX0DPKzN0VxFKlnQydZRLMs8K/Qwyu3iofBBGjKgGVwnQz0euZBDpN+gPufwaFKrEyKm2L+q18q8Bo1T5m9Q/hUQaYEDp11aCpWiCVAay3fKZ8Q1cNWmxuFW2QRzRG4uljSxLNNsjw6/S/FqOxlDYK+ieAXB7bShgyIFDFIsVgNM+9d1laRgYyyjTDpkWlEIWexbA1oYCq3hAXQq4RsCiIHbYT1p9j/EaQaQ6yy0OuYdOXtYxzCit6NR7Z7xvD/gJb0DnDHVQFTbrVlUpGd5m4IdqvKpDDLEYhO0HM5fl1dviRhA7sNc7bKpPrOqtJrrB2C07rMn1uRfeDGKJKYzR7wRyZY5yLDHJdJNlxfHcl3nYTejxnKTeqFX3X0GcmDGFZnLUhI7DRqWaXefRCABEZ9208so2WH2v7YdgAAAYxSURBVBqH4e+jf+lR7HsKg9w146bC+0GrmRbQ6yU7pZbjNPn/MXAzeBfbsMRNlnwX5j0VHu9FmAIGnGWyLRjTewXCOcfErOK+f8gANRqh8GmEKbq8BSbbY74rQt9kFGbatpcJGLN2w3ur4B4qGRvl/VrbopSRS3XCtnsKHqsUrB15ZDJBV2B0YlC4JCuUVjKqtqIlKPEAz1puZBwb6Muk1UoeRsYpTNiAbe+1CFw2lq2w1V+mGAptrRHm9BjihrY48k4OMjk0Tq14jy9smzTGNu239K2C8WxGqQtoxyHaHMQDOoOMthuVpLZW+Za7pZQStes/MRLT5jXn26RPNHk28Lz1eMJnafMck1C0i1stzambe8UwccRjex39K8eYKWNfuZxZdvIfMj1S3Gs5Y7ocOXYYzmGc8T48Jv4owIIqw5JTkuygodFcQCenbYcTwIg8sDrfCZuv2/C0KomPNdr8Pc1z/8ayALqRU4cBgD7m/0q2tYfQxfN8dpoxTOQZv0R+x/luCH3NtRLPgO0EVDGxDnnnsaMqog0iYZ9nfH/GXOmQgUt5zUFC9JUaFVcT1Mkjy/4WOXMLryXS4RI8gEs0ZszIiBOw1ucQ5h7u3Qm3qdh/lOclhh+VEc1bYHMeYXRbVv4+FP0tYgANeE5CIdF2cMZQXD0NJAPF/ornDfKjeMg9CzanItxpg5lu4jVB84j0Q+imKrNqog/1xkGRSN86rXaxBwPyIYuElGOOfibR748N9lxb/5MYXKENq7JEoIbVTPgTyFinZvfpi8imsxjLh0bOcxVDH0bZ+6xyZNbAG6cZzyPcfx7D+NSSknX6ncrkfZcJLAy/VtqcjZEUn8NSQ/idscRU0RDGGByX4s56vdu204JiVzrCTwxeKZ7vbUUW3Wwfh2hDBnJ9ATx/J5Pbq1gqDN1jAD0r5f6CiL/F2LzH2IS5t5BWHBhBRuyOEQMlmOGNMSRdETH92Oq+R9hRjdEOQX5VWc7mEeZ1EEMWsNK3OnZRZYz7bd5PY553kyK2wALtxiyZZ0yaAa/mnvd5rc1KQ5sY36dR0G2DzG+lnv23gcv/BvRev9y4iaTiEULSNu4Qr4kR/qLFL8RUr5Pa2zR6P8FgwUdXoBBDeCJhlHmcVbweIWxgcuRgmGLpi9A65JnN4CEoL28jK2s27TjPMxKNz3MZq5K8pxnamGtbo2r6eZ2BH2EgbqFsBy3/ataIe7ssKJpmyBQ6ZRxDlsJpO2n3bacvH1igXPLv5/8w3xPi6tso5WX6UWc5UEpL+ZxxE87ZDeS0xdBg6hmHZ/SpkfcPMEluMvnLGedEiugnaV+KwU01MfGKbfIfRE41lhelWt4gE30rY7ia8VvgfgI3yOL543yvBNle5PcSy3FbyQROw0AqXtqI8cmzU8Ry+rEbGZ0wOjwRDAUt7JKNTE4yLkE7SAsxdkrrCFkunJJWEy29Q1iAo5bG1Eb/kmiX+CiEUB255vj/pQE7/Ivpm6odrrE4qdZXfApXDAi103iGy9GfvcYVK36FKcu968ap2MN9mvGeQ7aVDRgsVoB5JWzCWQxciDaInOqRHRC6gfsaCGZ8VDnEm14x5rmJCGMFBmA5BqvQrHgAY7HDSFeGrKD9skHBZKHAJ1CuvzU28Wme/W9MgErbpu6yYPEzK0c6yMSSl/EL24JkshJ832oQM5gMubaXDzNIYyhwGQNxm2cqq38NA7mTAepEGbRyXjeCZAVRN9Dns1by8gDjtMZOAlWtUMcCoNytWfMoH2I896DAv7BKAOUYKcfuPEnNglNqYDyvMtGKUfB0K7YXrPggYxBpx59bftYVgwJKtAVDJViiNDwMtHyYcdvFuCWhI63IpBpehmW0qcAOXW4g93Ky+1UvK49QCddr7RAgiNw+xfiookAw5AKRuELbhvH+A4a4kWhyWIWBOmdxs7eZC2dsvMN8biXybTT6yikL+SQQhkkx6PIw/VKpX5+dNmv3NY18nhsUfMSQ6aAjIoNIGyP6LvzAyC7oH3k9okMRWSptRvl4kWcoiT0Sfoj8ryTryLMjfRavcEQ3I8+LjGPkfkKNEVacYqtKS4ksJBG9WAzIMqKvkcXSuVzEgufXq7Dh/ucKBAL/Bb6rs7VzGlc7AAAAAElFTkSuQmCC"/>
```

Comment:
Since these splatter backgrounds are basically png files base64 inlined inside of SVGs, it would probably be better to just have them as png background images.

Reply:
Got it. Using png files now.